web/src/components/Instruction.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
```

Comment:
Oh, yes!  Sorry, I guess I wasn't paying enough attention when I opened this PR - I'm not sure that's ripe enough to put in yet.  On the other hand it might on our side help when James dives into #1088, if we've already got a component that supports the new structure.

Would you rather take this component back out for now until it gets a proper design iteration?